maps/maps.go

```diff
@@ -190,10 +190,11 @@ func Search(mp map[string]interface{}, path []string) interface{} {
 // map[string]interface{} and not map[interface{}]interface{}.
 // Use IntfaceKeysToStrings() to convert if necessary.
 func Copy(mp map[string]interface{}) map[string]interface{} {
```

Comment:
The doc still has the old description of the method (using JSON marshal+unmarshal pass), can we update this.

Reply:
done.